sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -398,6 +400,18 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class Delay
+    {
+        protected Delay(System.TimeSpan? maxDelay = default(System.TimeSpan?), double jitterFactor = 0.2) { }
```

Comment:
nit: does "factor" come from something specific?  Why not just `jitter`?

Reply:
Yes, it comes from the fact that it is the multiplicative "factor" that is used against the delay to determine the jitter. So 0.2 means that for a delay of 10s, the jittered delay could end up being anywhere from 8s to 12s.